test/release/examples/benchmarks/shootout/chameneosredux.chpl

```diff
@@ -74,6 +74,12 @@ proc simulate(numChameneos) {
   // print information about the population once we're done
   //
   printInfo(chameneos);
+
+  //
+  // hasta las chameneos, baby.
+  //
+  destroyChameneos(chameneos);
+  delete meetingPlace;
```

Comment:
Yes, this change can be ported to master.
There should be no harm in porting the change in iterator.cpp to master as well.  (AMM on string-as-rec depends upon the change but it should have no visible effect on master.)

You can create a PR with one or both changes, if you like, or just leave it to me to port to master eventually.
(Creating a patch is pretty easy using git cherry-pick....)  If you do, you might choose to make the comment less jocular.  "hasta las chameneos" means literally "Until the fictitious deer-like animals", which doesn't make grammatical sense, for a start.

Let me know which way you'd like to go on this.
THH
________________________________
From: Lydia Duncan [notifications@github.com]
Sent: Wednesday, June 17, 2015 11:28 AM
To: chapel-lang/chapel
Cc: Tom Hildebrandt
Subject: Re: [chapel] String as rec June 11 leak fixes. (#2067)


In test/release/examples/benchmarks/shootout/chameneosredux.chpl<https://github.com/chapel-lang/chapel/pull/2067#discussion_r32658574>:

> @@ -74,6 +74,12 @@ proc simulate(numChameneos) {
>    // print information about the population once we're done
>    //
>    printInfo(chameneos);
> +
> +  //
> +  // hasta las chameneos, baby.
> +  //
> +  destroyChameneos(chameneos);
> +  delete meetingPlace;


Tom, does a similar change need to be done to the tests in test/studies/shootout/chameneos/bradc and /lydia? That's where this test comes from in the first place

—
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/2067/files#r32658574>.